src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,58 @@
+package racingcar.model;
+
+import racingcar.validator.CarValidator;
+
+import java.util.Objects;
+
+public class Car implements Comparable<Car> {
+    public static final int INITIAL_POSITION = 0;
+    public static final int MOVEABLE_POWER = 4;
+    public static final String COLON = " : ";
+    public static final String MOVE_BAR = "-";
+
+    private final String name;
+    private int position;
+
+    public Car(String name) {
+        this.name = name;
+        this.position = INITIAL_POSITION;
+        CarValidator.validateCar(name);
+    }
```

Comment:
필드에 값을 할당하기 전에 검증을 수행하는 게 더 자연스럽지 않을지.. 의견 드립니다 !

```suggestion
    public Car(String name) {
        CarValidator.validateCar(name);
        this.name = name;
        this.position = INITIAL_POSITION;
    }
```